conda-recipe/conda_build_config.yaml

```
@@ -0,0 +1,12 @@
+numpy:
+    - '1.25'
```

Do we need to align with `pyproject.toml`?
```suggestion
    - '1.26.4'
```

---

good point, if we're specifying `numpy>=1.26.4` in `pyproject.toml` we may as well do it here